### ISSUE TITLE
Creates nested <network> directory instead of using existing one

### ISSUE THREAD

**Issue by pinheadmz** [bcoin-org/bmultisig#25]
This is my first time playing with `bmultisig` so I could be using it wrong...
Seeing unexpected behavior in its data directory creation.

I'm running it with `--prefix= .../bcoin/regtest` and it is correctly finding the `wallet.conf` file in that location. (ports and API keys are parsed correctly and it seems to be running fine)
but then, unlike bwallet, it creates a NEW directory `regtest` INSIDE its prefix directory:

![screen shot 2018-12-07 at 8 40 50 am](https://user-images.githubusercontent.com/2084648/49660924-a29be280-f9fc-11e8-99c6-a0089d2060d9.png)

**Comment by nodar-chkuaselidze** [comment]
Yes, this behavior is actually coming standalone bwallet. Bmultisig does not actually do configuration parsing by itself it relies on bwallet to populate directories as well as pass options to bmultisig.

Something worth fixing in `bcoin`.

**Comment by pinheadmz** [comment]
Related: https://github.com/bcoin-org/bcoin/issues/645
